src/content/app/genome-browser/components/browser-sidebar-modal/modal-views/BookmarksModal.tsx

```diff
@@ -55,16 +57,22 @@ export const PreviouslyViewedLinks = () => {
           focus: buildFocusIdForUrl(previouslyViewedObject.object_id)
         });
 
+        const isFocusCurrentlyActive =
+          focusObjectId === previouslyViewedObject.object_id;
+
+        const onClick = isFocusCurrentlyActive
+          ? (event: React.MouseEvent<HTMLAnchorElement>) => {
+              changeFocusObject(focusObjectId);
+              event.preventDefault();
+            }
+          : () => onLinkClick(previouslyViewedObject.type, index);
+
         return (
           <div
             key={previouslyViewedObject.object_id}
             className={styles.linkHolder}
           >
-            <Link
-              replace
-              to={path}
-              onClick={() => onLinkClick(previouslyViewedObject.type, index)}
-            >
+            <Link replace to={path} onClick={onClick}>
```

Comment:
This is correct here. I thought this is ViewInApp component where `Link` will not be needed.